addons/web/static/src/legacy/scss/base_frontend.scss

```diff
@@ -1,5 +1,5 @@
 // Frontend general
-html, body, #wrapwrap {
+html:not(:has(.o_in_iframe)), body:not(.o_in_iframe), #wrapwrap {
```

Comment:
Back from leaves, just catching up on what I have been missing. Just a FWI here (I know it has been reverted): https://caniuse.com/css-has. `:has` can only be used in jQuery selectors, not CSS. Also the "o_in_iframe" class is probably an editor-only class, right? It should never appear in a css files for visitors.

Reply:
Thanks for the feedback and sorry for the temporary inconvenience! What do you think of this alternative fix: https://github.com/odoo/odoo/pull/83088 ?